jupyter/minimal/ubi8-python-3.8/test/test_notebook.ipynb

```diff
@@ -0,0 +1,65 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import unittest\n",
+    "import jupyterlab as jp\n",
+    "from platform import python_version\n",
+    "import nbdime\n",
+    "import nbgitpuller\n",
+    "\n",
+    "def get_major_minor(s):\n",
+    "    return '.'.join(s.split('.')[:2])\n",
+    "\n",
+    "class TestPythonVersion(unittest.TestCase):\n",
+    "    def test_version(self):\n",
+    "        expected_major_minor = '3.8'\n",
+    "        actual_major_minor = get_major_minor(python_version())\n",
+    "        self.assertEqual(actual_major_minor, expected_major_minor, \"incorrect version\")\n",
+    "\n",
+    "class TestDependenciesVersions(unittest.TestCase):\n",
+    "    def test_jupyter_version(self):\n",
+    "        expected_major_minor = '3.5'\n",
+    "        actual_major_minor = get_major_minor(jp.__version__)\n",
+    "        self.assertEqual(actual_major_minor, expected_major_minor, \"incorrect version\")\n",
+    "\n",
+    "    def test_nbgitpuller_version(self):\n",
+    "        expected_major_minor = '1.1'\n",
+    "        actual_major_minor = get_major_minor(nbgitpuller.__version__)\n",
+    "        self.assertEqual(actual_major_minor, expected_major_minor, \"incorrect version\")\n",
+    "\n",
+    "    def test_nbdime_version(self):\n",
+    "        expected_major_minor = '3.1'\n",
+    "        actual_major_minor = get_major_minor(nbdime.__version__)\n",
+    "        self.assertEqual(actual_major_minor, expected_major_minor, \"incorrect version\")\n",
+    "\n",
+    "unittest.main(argv=[''], verbosity=2, exit=False)"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.10.0"
```

Comment:
we are on python 3.8 for the notebook.
and the version is 3.10, we can rewrite for 3.8 ?

Reply:
I believe this metadata originates from the development environment where the test notebook was initially created. If you copy the content of the notebook run and save it within the corresponding notebook image, I believe this metadata should be resolved.